### PR TITLE
Added effect Spinning Cam

### PR DESCRIPTION
**Pull request by ProfessorBiddle**
Self explanitory, This one is a bit evil

Possible alternate name Vomit Comet

https://drive.google.com/file/d/1Kyy_yU7s8ysP7V7-vE7wITCZRRlvNESG/view

### CODE REVIEW

File: ChaosMod/Effects/db/Player/PlayerSpinCam.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/*
+    Effect by ProfessorBiddle, based on code from DrUnderscore, Idea from Synfetic
+*/
+
+#include <stdafx.h>
+static Camera flippedCamera = 0;
```
Comment (Last0xygen) [inline ChaosMod/Effects/db/Player/PlayerSpinCam.cpp:6]: Rename please ;P

**Review by Last0xygen — CHANGES_REQUESTED** [review]

**Comment by TheRealAyCe** [comment]
Could also include the character occasionally vomiting ( https://github.com/gta-chaos-mod/ChaosModV/issues/559 ) in this effect.

**Comment by kolyaventuri** [comment]
oh this is horrible.

I love it.

**Comment by ProfessorBiddle** [comment]
> Could also include the character occasionally vomiting ( #559 ) in this effect.

Not sure how to do that, if someone else knows that would be amazing

**Comment by OHSHIET** [comment]
Hmm I actually meant around the player, but this looks good too lol

**Comment by Last0xygen** [comment]
whats that glitch at 0:18? 🤔

**Comment by ProfessorBiddle** [comment]
> whats that glitch at 0:18? 🤔

not sure, probably something with the garbage ingame screen capture thing.

File: ConfigApp/Effects.cs [hunk 2]
```diff
@@ -576,6 +577,7 @@ public enum EffectType
             {EffectType.EFFECT_VEHS_CRUMBLE, new EffectInfo("Crumbling Vehicles", EffectCategory.VEHICLE, "vehs_crumble", true, true)},
             {EffectType.EFFECT_MISC_FPS_LIMIT, new EffectInfo("Console Experience", EffectCategory.MISC, "misc_fps_limit", true, true)},
             {EffectType.EFFECT_META_NO_CHAOS, new EffectInfo("No Chaos", EffectCategory.META, "meta_nochaos", true)},
+            {EffectType.EFFECT_PLAYER_SPIN_CAMERA, new EffectInfo("Spinning Camera", EffectCategory.PLAYER, "player_spincam", true)},
```
Comment (pongo1231) [inline ConfigApp/Effects.cs:580]: Doesn't match the effect id in the mod (`player_spinning_camera`)

**Pull request merged**